apps/condo/domains/news/components/NewsForm/CreateNewsForm.tsx

```diff
@@ -79,22 +82,33 @@ export const CreateNewsForm: React.FC = () => {
     const organizationId = useMemo(() => get(organization, 'id'), [organization])
 
     const createNewsItem = NewsItem.useCreate({ organization: { connect: { id: organizationId } } })
-    const action: BaseNewsItemFormProps['action'] = useCallback(async (values) => {
+    const action: BaseNewsFormProps['action'] = useCallback(async (values) => {
         return await createNewsItem(values)
     }, [createNewsItem])
 
     const {
         loading: isNewsItemTemplatesFetching,
         objs: newsItemTemplates,
         error: newsItemTemplatesError,
-    } = NewsItemTemplate.useObjects({})
+    } = NewsItemTemplate.useObjects({
+        where: {
+            OR: [
+                { organization_is_null: true },
+                { organization: { id: organizationId } },
+            ],
+        },
```

Comment:
This will work correctly, but these filters are already set within access rules (see `access/NewsItemTemplate.js`).

Reply:
Yes, now I understand. So can you please tell what you meant in your earlier comment

https://github.com/open-condo-software/condo/pull/3397#discussion_r1208893847

Reply:
The access rules filter models that the user has access to. In the case of a staff user, this filter returns only models related to the current user. But in the case of admin & support, we need to add an additional filter at the front end to display models for the selected organization. At the same time, we keep access of admins to all models using admin UI.

Reply:
In other words, the access rules will return a filter for all models for admin. To prevent loading templates from all organizations we add an additional filter at the front end. Firstly keystone calculates filter by access rules and secondly, it applies the filter got from a query.